boards/stm32f3discovery/Makefile

```diff
@@ -0,0 +1,26 @@
+# Makefile for building the tock kernel for the STM32F3DISCOVERY platform
+#
+TARGET=thumbv7em-none-eabi
+PLATFORM=stm32f3discovery
+
+include ../Makefile.common
+
+OPENOCD=openocd
+OPENOCD_OPTIONS=-f openocd.cfg
+
+# OpenOCD requires fullpath
+CWD=$(shell pwd)
+
+# sudo is needed for libusb access to work properly
+
+.PHONY: flash-debug
+flash-debug: target/$(TARGET)/debug/$(PLATFORM).elf
+	sudo $(OPENOCD) $(OPENOCD_OPTIONS) -c "init; reset halt; flash write_image erase $(CWD)/$<; verify_image $(CWD)/$<; reset; shutdown"
```

Comment:
This makes me nervous. I think we should remove the sudo from the flash commands, and point people to https://elinux.org/Accessing_Devices_without_Sudo instead.

Reply:
Sorry for that, I based my implementation on the nucleo boards, they both have it. I removed the sudo.